src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/sendtofirsttier/SendToFirstTierSubmittedHandler.java

```diff
@@ -49,6 +49,7 @@ public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
         if (isHandleCcdCallbackMapV2Enabled) {
             Optional<SscsCaseData> sscsCaseDataOptional = ccdCallbackMapService.handleCcdCallbackMapV2(
                     caseData.getPostHearing().getSendToFirstTier().getAction(),
+                    null,
```

Comment:
I suppose need a mutator instead of null as this would cause null pointer exception 
`    public SscsCaseDetails updateCaseV2(Long caseId, String eventType, String summary, String description, IdamTokens idamTokens, Consumer<SscsCaseData> mutator) {
        return this.updateCaseV2(caseId, eventType, idamTokens, (data) -> {
            mutator.accept(data);
            return new UpdateResult(summary, description);
        });
    }`

Reply:
Looks like a good spot - may need to change the sscs-common code....

Reply:
yes, I did that and it worked... but I think we should make that mutator Non-nullable in the commons code

Reply:
Made the change we suggested